osu.Game/Graphics/Cursor/MenuCursor.cs

```diff
@@ -97,13 +98,14 @@ protected override bool OnMouseDown(MouseDownEvent e)
 
         protected override void OnMouseUp(MouseUpEvent e)
         {
-            if (!e.IsPressed(MouseButton.Left) && !e.IsPressed(MouseButton.Middle) && !e.IsPressed(MouseButton.Right))
+            // cursor should go back to original size when none of main buttons are pressed
+            if (!(e.IsPressed(MouseButton.Left) || e.IsPressed(MouseButton.Middle) || e.IsPressed(MouseButton.Right)))
```

Comment:
I'd factor this out to a static method too (`anyMainButtonPressed(MouseEvent e)`). Can remove the comment at that point as the code will become self-documenting.